__tests__/app.test.js

```diff
@@ -213,131 +213,289 @@ describe('PATCH commands', () => {
+                      'https://images.pexels.com/photos/4473494/pexels-photo-4473494.jpeg?w=700&h=700',
+                    review_body: 'Fiddly fun for all the family',
+                    category: 'dexterity',
+                    created_at: "2021-01-18T10:01:41.251Z",
+                    votes: 5,
+                    comment_count :"3"
+                  })                
+            })
         });
+
+    test('200: should return reviews with category euro game', () => {
+        return request(app).get("/api/reviews?category=euro+game")
+        .expect(200)
+        .then(({body})=>{
+            const reviews = body.reviews;
+          
+            expect(reviews).toHaveLength(1);
+            expect(reviews[0]).toEqual({
+                    review_id : 1,
+                    title: 'Agricola',
+                    designer: 'Uwe Rosenberg',
+                    owner: 'mallionaire',
+                    review_img_url:
+                        'https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700',
+                    review_body: 'Farmyard fun!',
+                    category: 'euro game',
+                    created_at: "2021-01-18T10:00:20.514Z",
+                    votes: 1,
+                    comment_count :"0"
+                  })                
+            })
     });
 
-    test('404: GET followed by invalid review datatype should return a message', () => {
-        return request(app)
-        .get("/api/reviews/bananas")
-        .expect(400)
+    test('200: should return reviews with category social deduction', () => {
+        return request(app).get("/api/reviews?category=social+deduction")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("Bad Request - expected a number and got text e.g. received three instead of 3")
+            const reviews = body.reviews
+            expect(reviews).toHaveLength(11);
+            reviews.forEach((review)=>{
+                expect(review.category).toBe("social deduction")
+            })
         })
     });
 
-    test('400: GET follow by invalid review id ie resource that doesn\'t exist should return a message', () => {
-        return request(app)
-        .get("/api/reviews/99999")
-        .expect(404)
+    test('200: should return no reviews with category children\'s games', () => {
+        return request(app).get("/api/reviews?category=children's+games")
+        .expect(200)
         .then(({body})=>{
-            expect(body.message).toEqual("review id does not exist")
-        });
+            const reviews = body.reviews
+            expect(reviews).toHaveLength(0);
+        })
     });
 
-    test('400: GET followed by a comment on an invalid review datatype should return a message', () => {
-        return request(app)
-        .get("/api/reviews/bananas/comments")
-        .expect(400)
+    test('200: should return all reviews sorted by review_id ascending', () => {
```

Comment:
Same for this, you only need one test to prove all sortBys

Reply:
Please also write a test for default behaviour if no sort_by is given.

Reply:
This is done by review id test on line 33 already so didn't want to repeat myself

Reply:
Fair enough!